### PR TITLE
Clarify shortcomings of not having MachineDeployments

### PR DESCRIPTION
**Pull request by alvaroaleman**

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -145,6 +145,8 @@ Yes, though consider carefully before doing so.  Inspect the Match Labels from t
 ## Upstream kubernetes cluster-api project is utilizing MachineDeployments, why isn’t OpenShift?
 Any change to a MachineDeployment will result in an immediate removal and replacement of an entire MachineSet.  This is a much more costly operation that making changes to an instance in-place.  In particular RHEL CoreOS allows the VM to boot into an entirely updated operating system without having to perform a reinstallation.
 
+Modifications to a machine that can not be rolled out in-place like for example a change to the instance type must be rolled out manually by removing machines with the old config one by one.
```
Comment (enxebre) [inline FAQ.md:148]: or scaling down/up up/down depending on your scaling down strategy of choice.

Comment (alvaroaleman) [reply FAQ.md]: Updated

Comment (alvaroaleman) [reply FAQ.md]: @enxebre is there btw a RFE for this already so there is something we can track? If not, could you point me to where I can create such a RFE?

File: FAQ.md [hunk 2]
```diff
@@ -145,6 +145,9 @@ Yes, though consider carefully before doing so.  Inspect the Match Labels from t
 ## Upstream kubernetes cluster-api project is utilizing MachineDeployments, why isn’t OpenShift?
 Any change to a MachineDeployment will result in an immediate removal and replacement of an entire MachineSet.  This is a much more costly operation that making changes to an instance in-place.  In particular RHEL CoreOS allows the VM to boot into an entirely updated operating system without having to perform a reinstallation.
 
+Modifications to a machine that can not be rolled out in-place like for example a change to the instance type must be rolled out manually by scaling the MachineSet down and up again or by deleting machines one by one to trigger
```
Comment (elmiko) [inline FAQ.md:148]: small grammar nit

```suggestion
Modifications to a machine that can not be rolled out in-place (for example a change to the instance type) must be rolled out manually by scaling the MachineSet down and up again or by deleting machines one by one to trigger
```

**Comment by alvaroaleman** [comment]
@elmiko updated

**Pull request merged**